packages/11ty/_includes/components/figure/annotations-ui/index.js

```diff
@@ -21,15 +21,16 @@ module.exports = function(eleventyConfig) {
       const nameParts = [figure.id, index]
       if (lightbox) nameParts.unshift('lightbox')
       const name = nameParts.join('-')
-      const options = 
-        items.map((annotation, index) => figureOption({ annotation, index, input, name }))
+      const options = items
+        .map((annotation, index) => figureOption({ annotation, index, input, name }))
+        .join('\n')
       return html`
         <fieldset class="annotations-ui">
           <legend>${title}</legend>
           ${options}
         </fieldset>
       `
-    })
+    }).join('\n')
```

Comment:
a few things were not being joined, potentially leading to stray `,` gremlins

Reply:
odd, perhaps when we refactor at some point we can remove the need for any chained `join('\n') method calls?